Hazelnut/src/EditorLayer.cpp

```diff
@@ -36,6 +36,7 @@ namespace Hazel {
 
 		m_EditorScene = CreateRef<Scene>();
 		m_ActiveScene = m_EditorScene;
+		m_SceneHierarchyPanel.SetContext(m_ActiveScene);
```

Comment:
```suggestion
		NewScene();
```
This would require the changes to NewScene() proposed in #563.